packages/web/components/complex/assets-page-v1.tsx

```diff
@@ -60,13 +60,13 @@ export const AssetsPageV1: FunctionComponent = observer(() => {
       {
         label: t("assets.table.depositButton"),
         onClick: () => {
-          startBridge("deposit");
+          startBridge({ direction: "deposit" });
```

Comment:
I'm not opposed to objects as params, but is change actually necessary to make? I find the params being used are pretty straightforward and obvious, and there's no use of any optional/default params.

Reply:
What I like about using object parameters is that they are explicit and much easier to extend. If we ever need to add another property to the function, it's very straightforward to update it throughout the code